pkg/flare/archive.go

```diff
@@ -82,52 +83,55 @@ func createArchive(zipFilePath string, local bool, confSearchPaths SearchPaths,
 		// Only zip it up if the agent is running
 		err = zipStatusFile(tempDir, hostname)
 		if err != nil {
-			return "", err
+			log.Errorf("Could not zip status: %s", err)
 		}
 	}
 
-	err = zipLogFiles(tempDir, hostname, logFilePath)
-	if err != nil {
-		return "", err
-	}
-
 	err = zipConfigFiles(tempDir, hostname, confSearchPaths)
 	if err != nil {
-		return "", err
+		log.Errorf("Could not zip config: %s", err)
 	}
 
 	err = zipExpVar(tempDir, hostname)
 	if err != nil {
-		return "", err
+		log.Errorf("Could not zip exp var: %s", err)
 	}
 
 	err = zipDiagnose(tempDir, hostname)
 	if err != nil {
+		log.Errorf("Could not zip diagnose: %s", err)
 		return "", err
```

Comment:
Did you miss this one or is there a reason to return here ?

Reply:
it's a miss